python/test/unit/language/test_core.py

```diff
@@ -5619,3 +5619,86 @@ def kernel(
     kernel[(1, )](x_tri, y_tri, shape[0], BLOCK_SIZE=shape[0])
     # compare
     np.testing.assert_allclose(y_ref, to_numpy(y_tri), rtol=0.01)
+
+
+# -----------------------
+# test view slice
+# -----------------------
+
+view_layout = [
```

Comment:
These two are only used in the following test? Can you put them in the test?

Reply:
They are used to parameterize the test so I left them outside the test

Reply:
ah okay makes sense.